scripts/ldapSync.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -eu
+
+default_time_out=39600
+value_time_out=${SERVER_LDAP_SYNC_FULL_CRONJOB_TIMEOUT:-$default_time_out}
+
+default_api_key="example"
+value_api_key="${SYNC_API_KEY:-$default_api_key}"
+
+# Start server in the background and redirect logs to file
+nohup npm run nest:start > server.log 2>&1 &
+
+# Show logs and wait for server to start up
+echo "Server logs:"
+tail -f server.log &
+
+echo "Wait for server to start..."
+until curl -s -w --retry 360 --retry-connrefused --retry-delay 10 http://localhost:3030/serverversion >/dev/null 2>&1; do
+    sleep 1
+    echo "asleep"
+done
+
+# Start sync
+echo "Starting"
+curl --max-time $value_time_out -H "X-API-Key: $value_api_key" "http://localhost:3030/api/v1/sync?target=ldap&forceFullSync=true" | python3 -m json.tool
```

Comment:
not sure how authentication is handled, but at least is no change compared to before...

Reply:
is that just an opinion or which improvement are you referring to?

Reply:
just a thought / comment :)